sonar/modules/documents/jsonschemas/documents/document-v1.0.0_src.json

```diff
@@ -1298,19 +329,15 @@
           },
           "startDate": {
             "title": "Start date of publication",
-            "description": "Start date of the publication. This must be an integer, ie 1989, 453, -50. Used to sort search results. Once this field is set, a free formed date of publication can be added in the next field.",
             "type": "string",
-            "minimum": -9999,
-            "maximum": 2050,
-            "minLength": 1
+            "minLength": 1,
+            "pattern": "^[0-9]{4}(-[0-9]{2}-[0-9]{2})?$"
           },
           "endDate": {
             "title": "End date of publication",
-            "description": "End date of the publication. This must be an integer, ie 1989, 453, -50. Used to sort search results. Once this field is set, a free formed date of publication can be added in the next field.",
```

Comment:
Why did you abandon description?

Reply:
This will be added later, when the documents editor will be updated. The PO will give me the needed descriptions.